src/EditorFeatures/CSharpTest/EncapsulateField/EncapsulateFieldTestState.cs

```diff
@@ -30,31 +30,26 @@ internal class EncapsulateFieldTestState : IDisposable
         public Document TargetDocument { get; }
         public string NotificationMessage { get; private set; }
 
-        private static readonly IExportProviderFactory s_exportProviderFactory =
-            ExportProviderCache.GetOrCreateExportProviderFactory(
-                TestExportProvider.MinimumCatalogWithCSharpAndVisualBasic.WithParts(
-                    typeof(CSharpEncapsulateFieldService),
-                    typeof(EditorNotificationServiceFactory),
-                    typeof(DefaultTextBufferSupportsFeatureService)));
-
         public EncapsulateFieldTestState(TestWorkspace workspace)
         {
             Workspace = workspace;
             _testDocument = Workspace.Documents.Single(d => d.CursorPosition.HasValue || d.SelectedSpans.Any());
             TargetDocument = Workspace.CurrentSolution.GetDocument(_testDocument.Id);
 
+            // TODO: use mock INotificationService https://github.com/dotnet/roslyn/issues/46045
```

Comment:
📝 If #46045 is updated to say that `INotificationServiceCallback` should be removed at the same time, then this comment is unnecessary since the interface removal will force this to need an update.